File: microservices/frontend/web-app/src/core_test/brand/domain/BrandMother.ts
import {Brand} from "../../../core/brand/domain/Brand";
import {UUID} from "../../../core/shared/domain/UUID";

export class BrandMother {

    static randomBrand(): Brand {
        return {
            brandId: UUID.random(),
            name: 'Any Brand'
        };
    }

    static randomBrands(number: number): Brand[] {
        const brands: Brand[] = [];
        for (let i = 0; i < number; i++) {
            brands.push(BrandMother.randomBrand());
        }

        return brands;
    }
}

File: microservices/backend/catalog-service/core_test/brand/domain/brand_mother.py
from typing import List

from core.brand.domain.brand import Brand
from core.brand.domain.brand_id import BrandId
from core.shared.base.domain.uuid import UUID


class BrandMother:
    @staticmethod
    def random_id() -> BrandId:
        return BrandId(
            UUID.random()
        )

    @staticmethod
    def random_brands() -> List[Brand]:
        brand_list = [
            Brand(
                brand_id=BrandMother.random_id(),
                name='any_brand_1'
            ),
            Brand(
                brand_id=BrandMother.random_id(),
                name='any_brand_2'
            )
        ]

        return brand_list
